accessibility.html

```diff
@@ -0,0 +1,281 @@
+                <a href="images.html">4. Images</a><br />
+                <a href="index.html">5. Run Deployment</a>
+              </nav>
+            </aside>
+            <article role="article">
+              <h1>Accessibility</h1>
+              <p>
+                This is a guide to making everything at Hoodie accessible.
+              </p>
+              <ul>
+                <li><a href="#introduction">Introduction</a></li>
+                <li><a href="#factors">Factors affecting experiences with the web</a></li>
+                <li><a href="#guidelines">Accessibility guidelines</a></li>
+                <li><a href="#example">An example of making Hoodie accessible</a></li>
+                <li><a href="#tools">Accessibility testing</a></li>
+              </ul>
+            </article>
+          </section>
+
+          <section class="cb">
+            <article id="introduction" role="article">
+              <h2>Introduction</h2>
+              <p>
+                In 1997, the W3 launched the "International Program Office for Web Accessibility Initiative".
+                In the <a href="http://www.w3.org/Press/IPO-announce">press release</a>, Tim Berners-Lee said the following:
+                "The power of the Web is in its universality. Access by everyone regardless of disability is an essential aspect".
+              </p>
+              <p>
+                One of the first things you see when you go to the [Hoodie website] is the heading "Hoodie is for you".
+                By "you", we mean everyone. By everyone, we mean <em>everyone</em>. That brings us to this guide, which outlines how all contributors can make Hoodie accessible.
+              <p>
+            </article> 
+          </section>
+          
+          <section class="cb">
+            <article id="factors" role="article">
+              <h2>Factors affecting experiences with the web</h2>
+              <p>
+                A common misconception when we talk about accessibility is that we mean to cater for those who have long-term limited abilities or impairments <em>only</em>.
+                Often mentioned examples are blindness and motor and cognitive disabilities. It includes that, but it also includes so much more.
+              </p>
+              <p>
+                Here are some examples of things that can hinder and change your experience with the web:
+                <ul>
+                  <li>Blindness, either partial or total</li>
+                  <li>Colour blindness, of which there are many types</li>
+                  <li>Ataxia (loss of control of body movement). Each type affecting motor control in different ways</li>
+                  <li>Hearing loss, either partial or total</li>
+                  <li>Epilepsy</li>
+                  <li>Distraction</li>
+                  <li>Dyslexia</li>
+                  <li>Age</li>
+                  <li>Depression</li>
+                  <li>Tiredness</li>
```

Comment:
Is this common British English to say tiredness? In the USA it would probably be "fatigue". Tiredness doesn't feel like a word to me, but it could just be dialect.